packages/contentful--app-scripts/src/generate-function/clone.ts

```diff
@@ -78,17 +100,24 @@ export function resolvePaths(localPath: string) {
   return { localTmpPath, localFunctionsPath };
 }
 
-export async function cloneAndResolveManifests(cloneURL: string, localTmpPath: string, localPath: string, localFunctionsPath: string) {
+export async function cloneAndResolveManifests(cloneURL: string, localTmpPath: string, localPath: string, localFunctionsPath: string, keepPackageJson?: boolean) {
```

Comment:
Maybe change this to have a default of `false` for `keepPackageJson`?

```suggestion
export async function cloneAndResolveManifests(cloneURL: string, localTmpPath: string, localPath: string, localFunctionsPath: string, keepPackageJson = false) {
```